kodilla-patterns2/src/main/java/com/kodilla/patterns2/aop/calculator/Calculator.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.patterns2.aop.calculator;
+
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+
+@Component
+@EnableAspectJAutoProxy
```

Comment:
`@EnableAspectJAutoProxy` to jest adnotacja ktora wlacza w calej aplikacji aspectJ. Powinna byc w klasie konfiguracyjnej, nie w komponencie. 
Np w `KodillaPatterns2Application`